audits/c-dev-organization-2026-05-07.md

```
@@ -0,0 +1,195 @@
+#### Loose `.csv` (2 files)
+
+| File | Destination |
+|---|---|
+| `[CM] Dados CRM.csv` | `business-ai-first/db-inventory/exports/2026-05-07/` |
+| `[CM] Dados CRM_v2.csv` | Same |
+
+---
+
+## Proposed target structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)